XenModel/Messages.resx

```diff
@@ -4644,6 +4671,12 @@ Would you like to eject these ISOs before continuing?</value>
   <data name="ENABLE_PLAIN" xml:space="preserve">
     <value>Enable</value>
   </data>
+  <data name="ENABLE_PVS_READ_CACHING_RUBRIC_MULTIPLE" xml:space="preserve">
+    <value>Choose the PVS site where the selected VMs should be cached. You can also configure these VMs to be prepopulated in the PVS Cache.</value>

```

Comment:
Please remove the text that refers to prepopulation from ENABLE_PVS_READ_CACHING_RUBRIC_MULTIPLE and ENABLE_PVS_READ_CACHING_RUBRIC_SINGLE